FlowCrypt/Controllers/Settings/KeySettings/Key Detail Info/KeyDetailInfoViewDecorator.swift

```diff
@@ -36,7 +25,8 @@ struct KeyDetailInfoViewDecorator: KeyDetailInfoViewDecoratorType {
         case .keyWord:
             title = "key_settings_detail_key_words".localized + ":" + " "
             return title.attributed(.medium(16))
-                + keyId.keywords.attributed(.regular(16), color: .main)
+            // TODO: - Tom?
```

Comment:
@tomholub previously we had keywords in KeyDetailInfoViewCointroller. 
Should keywords be part of KeyDetail? Or this part should be removed?

Reply:
please remove keywords from all parts of the app. They are not useful, I'm deprecating them